azure-pipelines.yml

```diff
@@ -87,10 +110,10 @@ steps:
     addChangeLog: false
 
 - task: Bash@3
-  displayName: "Verify wheel"
+  displayName: "Install wheel"
   inputs:
     targetType: 'inline'
-    script: pip install --upgrade iotedgedev && iotedgedev --version
+    script: pip install dist/iotedgedev-${VERSION}-py3-none-any.whl && iotedgedev --version
```

Comment:
>script: pip install dist/iotedgedev-${VERSION}-py3-none-any.whl && iotedgedev --version [](http://example.com/codeflow?start=4&length=87)

this step should be right after l52 i think to validate the locally built wheel? Because this pipeline no longer publishes to pypi directly we cannot validate the install from pypi as it was the original intention.

Reply:
I have removed the Install wheel task and added it to the build wheel.